Library/Homebrew/utils/analytics.sh

```diff
@@ -1,30 +1,45 @@
 setup-analytics() {
-  # User UUID file. Used for Homebrew user counting. Can be deleted and
-  # recreated with no adverse effect (beyond our user counts being inflated).
-  HOMEBREW_ANALYTICS_USER_UUID_FILE="$HOME/.homebrew_analytics_user_uuid"
+  local git_config_file="$HOMEBREW_REPOSITORY/.git/config"
+
+  # Migrate analytics UUID to its new home in Homebrew repo's git config and
+  # remove the legacy UUID file if detected.
+  migrate_legacy_uuid_file () {
```

Comment:
Minor style note: [`cmd/update.sh`](https://github.com/Homebrew/brew/blob/b0d906f0f8d90c08b0b799d5cf0fd18bf78c30da/Library/Homebrew/cmd/update.sh) is using underscores/snake_case in its function names. Maybe we should change one of these to match the other?